.github/workflows/trufflehog-commit.yml

```diff
@@ -0,0 +1,18 @@
+name: Leaked Secrets Scan on Commits
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
Is this a literal branch name? IIRC os-autoinst-distri-opensuse does not have a main branch, but master.